health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/ConsoleToolTip.js

```
@@ -0,0 +1,71 @@
+    setTooltipPos(null);
+  };
+
+  return (
+    <div className="app-config-drawer-subheader">
+      <span
+        ref={iconRef}
+        onMouseEnter={showTooltip}
+        onMouseLeave={hideTooltip}
+        style={{ cursor: "pointer", display: "inline-block", marginLeft: 8 }}
+        className={`icon-wrapper ${className}`}
+      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)